packages/sanes-chrome-extension/src/logic/persona/persona.ts

```diff
@@ -11,6 +11,7 @@ import {
   pathBuilderForCodec,
 } from '../config';
 import { AccountManager, AccountInfo, AccountManagerChainConfig } from './accountManager';
+import { ProcessedTx } from '../../routes/account/components/ListTxs';
```

Comment:
A data class/file like Persona should not depend on a UI component to avoid circular dependencies. In this case this can easily be fixed by moving the data type `ProcessedTx` into `logic/persona/persona.ts`, where the `ProcessedTx` are created.

Reply:
Sounds good.